performance_test/helper_scripts/generate_apex_xml.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generate DEFAULT XML file for apex tests."""
+import argparse
+from xml.dom.minidom import parseString
+
+from dicttoxml import dicttoxml
+
+
+def item_names(parent_name):
+    """
+    Get the tag of an item based on its parent.
+
+    :param parent_name: The name of the parent.
+    :return: The item tag.
+    """
+    if parent_name == 'interfaceWhiteList':
+        return 'address'
+    elif parent_name == 'transport_descriptors':
+        return 'transport_descriptors'
+
+
+if __name__ == '__main__':
+    # Get arguments
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument(
+        '-w',
+        '--whitelist',
+        help='List of whitelist interfaces separated by comma with no spaces',
+        required=True,
+    )
+    parser.add_argument(
+        '-f',
+        '--file',
+        help='Path to the file to store the XML',
+        required=True,
+    )
+    args = parser.parse_args()
+    interface_whitelist = args.whitelist.split(',')
```

Comment:
Instead of calling split here, `argparse` has support for arguments that take multiple values, here's an example of how to do it:

https://stackoverflow.com/a/15753721/470581